src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -69,6 +69,24 @@
     optional = true)
     public boolean dropAboveGqThreshold = false;
 
+    @Argument(fullName = "write-reference-ranges",
+            shortName = "rr",
+            doc = "write reference ranges data",
+            optional = true)
+    public boolean writeReferenceRanges = false;
+
+    @Argument(fullName = "enable-vet",
+            shortName = "ev",
+            doc = "write vet data",
+            optional = true)
+    public boolean enableVet = true;
+
+    @Argument(fullName = "enable-pet",
+            shortName = "ep",
+            doc = "write pet data",
+            optional = true)
+    public boolean enablePet = true;
+
```

Comment:
maybe I'm misunderstanding---but what is the use case for enabling vet but not pet?